code/game/objects/items/plushes.dm

```diff
@@ -22,6 +22,7 @@
 	var/heartbroken = FALSE
 	var/vowbroken = FALSE
 	var/young = FALSE
+	var/divine = FALSE
```

Comment:
if you wanna stay on web editor you could just make a separate commit reversing all this plushie stuff

Reply:
Alright, I'll do that tomorrow morning.